### ISSUE TITLE
ReferenceError: dlls is not defined on setup

### ISSUE THREAD

**Issue by GlisboaDev** [storybooks/storybook#4984]
Hi,
I'm currently trying to add a storybook to our React Native project, but I'm getting the following error when trying to run it (yarn storybook):

```
yarn run v1.10.1
$ storybook start
=> Loading custom .babelrc from project directory.
=> Loading custom addons config.
=> Using default webpack setup based on "Create React App".

React Native Storybook started on => http://localhost:7007/

webpack built 4948fc70c66e637eaac7 in 3698ms
✖ ｢wdm｣: Hash: 4948fc70c66e637eaac7
Version: webpack 4.27.1
Time: 3698ms
Built at: 2018-12-12 15:19:29
                   Asset       Size   Chunks             Chunk Names
              index.html  473 bytes           [emitted]  
static/manager.bundle.js   6.73 MiB  manager  [emitted]  manager
Entrypoint manager = static/manager.bundle.js
[0] multi ./storybook/addons.js ./node_modules/@storybook/react-native/dist/manager/index.js 40 bytes {manager} [built]
[./node_modules/@storybook/addon-actions/dist/manager.js] 994 bytes {manager} [built]
[./node_modules/@storybook/addon-actions/register.js] 38 bytes {manager} [built]
[./node_modules/@storybook/addon-links/dist/manager.js] 1.11 KiB {manager} [built]
[./node_modules/@storybook/addon-links/register.js] 38 bytes {manager} [built]
[./node_modules/@storybook/addons/dist/index.js] 4.23 KiB {manager} [built]
[./node_modules/@storybook/channel-websocket/dist/index.js] 3.13 KiB {manager} [built]
[./node_modules/@storybook/core-events/index.js] 493 bytes {manager} [built]
[./node_modules/@storybook/mantra-core/index.js] 41 bytes {manager} [built]
[./node_modules/@storybook/podda/dist/index.js] 4.06 KiB {manager} [built]
[./node_modules/@storybook/react-native/dist/manager/index.js] 467 bytes {manager} [built]
[./node_modules/@storybook/react-native/dist/manager/provider.js] 5.78 KiB {manager} [built]
[./node_modules/@storybook/ui/dist/index.js] 3.52 KiB {manager} [built]
[./node_modules/global/window.js] 232 bytes {manager} [built]
[./storybook/addons.js] 88 bytes {manager} [built]
    + 691 hidden modules

ERROR in Template execution failed: ReferenceError: dlls is not defined

ERROR in   ReferenceError: dlls is not defined
  
  - index.ejs:47 eval
    [.]/[loader.js!.]/[@storybook]/core/src/server/templates/index.ejs:47:2
  
  - index.ejs:60 module.exports
    [.]/[loader.js!.]/[@storybook]/core/src/server/templates/index.ejs:60:3
  
  - index.js:407 templateParamsPromise.then
    [mobile]/[html-webpack-plugin]/index.js:407:16
  
  
  - next_tick.js:188 process._tickCallback
    internal/process/next_tick.js:188:7
  

Child HtmlWebpackCompiler:
                          Asset      Size               Chunks  Chunk Names
    __child-HtmlWebpackPlugin_0  1.39 MiB  HtmlWebpackPlugin_0  HtmlWebpackPlugin_0
    Entrypoint HtmlWebpackPlugin_0 = __child-HtmlWebpackPlugin_0
    [./node_modules/html-webpack-plugin/lib/loader.js!./node_modules/@storybook/core/src/server/templates/index.ejs] 1.75 KiB {HtmlWebpackPlugin_0} [built]
    [./node_modules/lodash/lodash.js] 527 KiB {HtmlWebpackPlugin_0} [built]
    [./node_modules/webpack/buildin/global.js] (webpack)/buildin/global.js 472 bytes {HtmlWebpackPlugin_0} [built]
    [./node_modules/webpack/buildin/module.js] (webpack)/buildin/module.js 497 bytes {HtmlWebpackPlugin_0} [built]
ℹ ｢wdm｣: Failed to compile.
 ```

relevant dependencies: 

```
    "react": "16.6.1",
    "react-native": "0.57.5",

    "@storybook/react-native": "^4.1.0",
    "@storybook/addon-actions": "^4.1.0",
    "@storybook/addon-links": "^4.1.0",
    "@storybook/addons": "^4.1.0",
```

**System:**
 - OS: MacOS Mojave 10.14.1
 - Storybook Version: 4.1.0 (also attempted with 4.0.12)
- React native version: 0.57.5

Steps to reproduce: 
I attempted to run using both script setup (`npx -p @storybook/cli sb init`) and manual as described on the Slow Start Guide, but got the same issue on both cases.

**Comment by igor-dv** [comment]
The fix is out with 4.1.1
Please try it out and let us know if it's working now for you.

**Comment by GlisboaDev** [comment]
Upgrading to `4.1.1` fixes the issue.
Thanks
